client/src/components/NewParkReviewForm.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react"
+import translateServerErrors from "../services/translateServerErrors"
+import ErrorList from "./layout/ErrorList"
+
+const NewParkReviewForm = (props) => {
+    const parkId = props.parkId
+    const defaultReview = {
+        rating: "",
+        reviewText: ""
+    }
+    const [newReview, setNewReview] = useState(defaultReview)
+    const [errors, setErrors] = useState([])
+
+    const postNewParkReview = async () => {
+        try {
+            const response = await fetch(`/api/v1/parks/${parkId}`, {
```

Comment:
if we are creating a new Review, then the resource `reviews` needs to appear in the POST request path - "api/v1/parks/parkId/reviews" to indicate that we are creating a new Review